documentation/src/main/asciidoc/advanced-features.asciidoc

```diff
@@ -19,6 +20,22 @@ SearchFactory searchFactory = fullTextSession.getSearchFactory();
 ----
 ====
 
+[[AccessingSearchIntegrator]]
+=== Accessing the SearchIntegrator
+The SearchIntegrator is an object that gives access to lower level API of Hibernate Search.
+Frameworks can use it to integrate with Hibernate Search. It should not be exposed as a
+public API.
```

Comment:
"It should not be exposed as a public API"  - not sure what this is supposed to mean. It sounds like a recommendation to the user to now expose it as public API, but that is really not in the hands of the users. Given its package name (`org.hibernate.search.spi`), I would just say it is a SPI.